src/services/widgets.data.service.ts

```diff
@@ -16,8 +16,9 @@ export const fetchWidgets = async (id: number, lang: string, yearAgo?: number):
     if (yearAgo) {
       query.push(`years_ago=${yearAgo}`);
     }
-    const widgetsUrl = `${NEWS_FLASH_API}?${query.join('&')}`;
-    //temp - long response time of server- keep console.log to see out url
+  const widgetsUrl = `${NEWS_FLASH_API}?${query.join('&')}${
+      process.env.REACT_APP_SHOW_MOCK_INFORMATION && `&mock=${process.env.REACT_APP_SHOW_MOCK_INFORMATION}`
```

Comment:
I think it's better to keep code in same style and more readable:
const isMock = process.env.REACT_APP_SHOW_MOCK_INFORMATION
if(isMock){
query.push(`mock=${isMock}`);
}